# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1495583502 25200
#      Tue May 23 16:51:42 2017 -0700
# Node ID ba25f4e38a230eedb104de7e7dd5a0d3bac783f6
# Parent  dc3745744aefd97cfeabd98f5d82eb08aa369535
# Parent  22ed4204b2b793dbc43b0bb5b00c5b93d45020b7
maint: Merge stable to default.

diff --git a/libgui/qterminal/libqterminal/unix/TerminalView.cpp b/libgui/qterminal/libqterminal/unix/TerminalView.cpp
old mode 100755
new mode 100644
diff --git a/libgui/qterminal/libqterminal/unix/TerminalView.h b/libgui/qterminal/libqterminal/unix/TerminalView.h
old mode 100755
new mode 100644
diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
old mode 100755
new mode 100644
